### PR TITLE
Add tests to cover new ASCVD algorithm with new vitals boundaries and optimal values. Fixes #18

### LINKED ISSUE #18: Add tests to accommodate for new ASCVD algorithm and UI
In order to fully support the new ASCVD algorithm and the new UI elements that come along with updating the Cardiac Risk App, we have to add/update tests to our existing test suite that test these new aspects of the app.

### PR DESCRIPTION
**Pull request by zplata**
Currently, the test suite in place for the Cardiac Risk App does not have tests accommodating the new ASCVD Risk algorithm or the updates to the UI. We can begin to add tests that cover the new algorithm in place and the adjusted optimal levels for some patient vitals to this suite. As the UI/UX of this app gets updated, more tests can be updated/added accordingly.

For easier readability, here are mostly the changes you are seeing:

- Modified `setPatientInfo` everywhere in the suite to accommodate new patient factors (ex: diabetic status, race, etc.)
- Added tests for computing the age of a patient
- Added tests for computing 10-year/Lifetime ASCVD Risk
- Modified several tests for the new optimal values and boundaries of values as defined in the [ASCVD Risk Estimator](http://tools.acc.org/ASCVD-Risk-Estimator/)
- Modified tests to check for grouper instead of direct Systolic Blood Pressure LOINC code
- Added tests for message displays in ASCVD computation

@kpshek 
@mjhenkes 
@kolkheang
@koushic88
@shriniketsarkar

### CODE REVIEW

**Comment by mjhenkes** [comment]
+1

File: build/js/cardiac_risk.js [hunk 1]
```diff
@@ -1140,6 +1143,32 @@
         CardiacRisk.graphData.hdlSliderData.upperBound);
   }
 
+/**
```
Comment (shriniketsarkar) [inline build/js/cardiac_risk.js]: Should this be part of the polyfill.js file ?

**Comment by shriniketsarkar** [comment]
👍

File: build/js/cardiac_risk.js (hunk 1 above)
Comment (zplata) [inline build/js/cardiac_risk.js]: It is already. I think the one thing I forgot in my last PR getting that file in was to include that src/js/cardiac_risk_polyfill.js in the grunt file to build it. I didn't need to worry about it until the adding of tests here.
